.pipelines/arc-register-extension-all-regions-stable.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+# Register azuremonitor-containers extension with Arc Registration API
+
+REGISTER_REGIONS_CANARY=$REGISTER_REGIONS_CANARY
+RELEASE_TRAINS_PREVIEW_PATH=${RELEASE_TRAINS_PREVIEW_PATH}
```

Comment:
add empty checks to avoid triggering release with empty release name(s)

Reply:
added